docu/sphinx/source/examples.rst

```diff
@@ -24,13 +24,15 @@ function is also capable of handling :code:`INF` singularities.
 
 .. literalinclude:: ../../examples/example1-plain.ipf
    :caption: example1-plain.ipf
+   :tab-width: 4
    :linenos:
    :emphasize-lines: 11
 
 The test suite can be executed using the following command:
 
 .. code-block:: igor
    :caption: command
+   :tab-width: 4
```

Comment:
The :tab-width:  option does not exist for .. code-block blocks.
These blocks have to be changed to literalinclude blocks for that.

Reply:
@MichaelHuth Good point. I've removed them from all code-blocks. Thinking about it is also does not make sense as we in these.